### PR TITLE
Adds Django REST Framework 3.4 compatibility

### PR DESCRIPTION
**Pull request by kciredor**

### CODE REVIEW

**Comment by kciredor** [comment]
Tests seem fine; py27-lint is not in scope of this PR.

**Comment by aleontiev** [comment]
:+1: I agree with your assessment, but let me spend some time trying to figure out why py27-lint is failing so we can get a clean build here.

File: README.md [hunk 1]
```diff
@@ -610,43 +610,56 @@ Not all versions of Python, Django, and DRF are compatible. Here are the combina
...
+| 3.5    | 1.8    | 3.4 | YES |
 | 3.5    | 1.9    | 3.1 | NO<sup>1</sup> |
 | 3.5    | 1.9    | 3.2 | YES |
 | 3.5    | 1.9    | 3.3 | YES |
+| 3.5    | 1.9    | 3.4 | YES |
 
 * 1: Django 1.9 is not compatible with DRF 3.1
 * 2: Django 1.9 is not compatible with Python 3.3
 * 3: Django 1.7 is not compatible with Python 3.5
+* 3: Django 1.7 is not compatible with DRF 3.4
```
Comment (aleontiev) [inline README.md]: This should be "**4**: Django 1.7 is not compatible with DRF 3.4"

**Comment by aleontiev** [comment]
@kciredor we've removed the isort linter, which was causing py27-lint to fail and seemed flaky (even on master).

If you pull in that change (https://github.com/AltSchool/dynamic-rest/pull/121) and fix the README.md typo, I'd be happy to merge this.

**Comment by kciredor** [comment]
@aleontiev tnx, all good! :+1:

**Comment by aleontiev** [comment]
:+1:

**Comment by aleontiev** [comment]
Merged, thanks for your contribution :)

**Pull request merged**